### PR TITLE
Update copyright

### LINKED ISSUE #2342: Update copyright?
```
 <Copyright>Copyright 2015-2017 Joseph T. McBride et al. All rights reserved.</Copyright>
```

Is it worth changing the year here? Something else? Ping @joemcbride

### PR DESCRIPTION
**Pull request by sungam3r**
fixes #2342

### CODE REVIEW

File: .github/workflows/test-code.yml
Comment (sungam3r) [inline .github/workflows/test-code.yml:63]: `-w, --fix-whitespace                                                       Run whitespace formatting. Run by default when not applying fixes.` 🤦‍♂️

**Comment by Shane32** [comment]
If we update the copyright, the license file should be updated, and @joemcbride should approve it.

**Comment by sungam3r** [comment]
Done.

**Review by Shane32 — APPROVED** [review]
@joemcbride to also approve

**Comment by Shane32** [comment]
@joemcbride Please approve when you have a chance.

**Review by joemcbride — APPROVED** [review]

**Pull request merged**
